.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Nuvem Development Container",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:22-bookworm",
```

Comment:
Can we use a fixed hash with a comment stating human friendly tag? See GitHub workflows for examples